lang/el.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: el\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Χρήση: vrms-rpm [επιλογές]"
+msgid "msg_help_option_ascii"
+msgstr "Εμφανίζει rms ASCII-art όταν δεν βρεθούνε  nonfree πακέτα, \ n"
+       "ή όταν τα nonfree πακέτα είναι 10 %% ή περισσότερο του συνόλου."
+msgid "msg_help_option_explain"
+msgstr "Κατά την καταχώριση των πακέτων, εμφανίστε τις άδειες ώστε να δικαιολογηθεί \ n"
+       "η free / nonfree ταξινόμηση."
+msgid "msg_help_option_help"
+msgstr "Εμφάνιση αυτής της βοήθειας και έξοδος".
+msgid "msg_help_option_list"
+msgstr "Εκτός από την εμφάνιση ενός συνοπτικού αριθμού free & nonfree - πακέτων, \ n"
+       "εκτυπώστε τα βάση ονόματος. Η προεπιλεγμένη τιμή είναι" nonfree "."
+msgid "msg_help_option_version"
+msgstr "Εμφάνιση πληροφοριών έκδοσης και έξοδος."
+msg_list_noargument "
+msgstr "Η επιλογή% s: --list απαιτεί ένα όρισμα"
+msg_list_badargument "
+msgstr "% s: Το όρισμα για την επιλογή --list πρέπει να είναι ένα από τα 'none', 'free', 'nonfree' ή 'all'"
+msgid "msg_unknown_option"
+msgstr "% s: άγνωστη επιλογή '% s'"
+msgid "msg_total_free"
+msgstr "% d ελεύθερα πακέτα (free)"
```

Comment:
`%d` is similar to `%s`, except it's replaced by a number. There must NOT be a space between the two characters.